inst/extdata/emissions/GFED_Deforest_BC.csv

```diff
@@ -0,0 +1,248 @@
+# File: GFED_Deforest_BC.csv
+# Title: BC Emissions from deforestation by country 
+# Units: unknown
```

Comment:
I looked at this a bit with Rachel, only the person who originally generated this data knows. I don't know who that was. However, its only used in this as a ratio between two of these numbers, to the units aren't important in this case.